stdlib/rdp/plot.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"math"
+	"math/rand"
+
+	"gonum.org/v1/plot"
+	"gonum.org/v1/plot/plotter"
+	"gonum.org/v1/plot/plotutil"
+	"gonum.org/v1/plot/vg"
+)
+
+func SavePlot(orig, simp plotter.XYs) error {
```

Comment:
Looks like this is code duplication. This method is already present int rdp.go file.

Reply:
This is an obsolete file and was intended to be part of this commit. Will remove them